dd-java-agent/appsec/src/main/java/com/datadog/appsec/gateway/GatewayBridge.java

```diff
@@ -164,6 +169,11 @@ public void init() {
                     });
               }
             }
+
+            // If extracted any Api Schemas - commit them
+            if (!ctx.commitApiSchemas(traceSeg)) {
+              log.warn("Unable to commit api security schemes");
```

Comment:
```suggestion
              log.warn("Unable to commit api security schemas");
```

What does this mean? When could it happen? Can users do something about it when it happens (AKA do we want an escalation in this scenario)?

Reply:
Switched to `debug`. This error could happen if trace segment or list of collected schemas are `null`. This case should never happen.